Added a number of 3D models for sip components

Misca1234 opened the pull request:
Added a number of 3D models

All are scripted

This push require an foot print update, foot print PowerIntegrations_eSIP-7F was wrong
wait to merge this push until that is pushed (coming)
https://github.com/KiCad/kicad-footprints/pull/1202


3D model push
https://github.com/KiCad/kicad-packages3D/pull/484

3D script push
https://github.com/easyw/kicad-3d-models-in-freecad/pull/244

![bild](https://user-images.githubusercontent.com/25547797/50358904-29070700-055b-11e9-9de4-41568183ff28.png)


![bild](https://user-images.githubusercontent.com/25547797/50358908-2e645180-055b-11e9-809e-8166644b90eb.png)

![bild](https://user-images.githubusercontent.com/25547797/50358922-33c19c00-055b-11e9-9aa2-fe5be24cd9a6.png)

![bild](https://user-images.githubusercontent.com/25547797/50358932-3ae8aa00-055b-11e9-8e5c-1769945382ff.png)

![bild](https://user-images.githubusercontent.com/25547797/50358944-43d97b80-055b-11e9-8da1-fa688a890cba.png)

![bild](https://user-images.githubusercontent.com/25547797/50358952-49cf5c80-055b-11e9-81c5-530aabb62c77.png)

![bild](https://user-images.githubusercontent.com/25547797/50358955-505dd400-055b-11e9-8f7a-a8535bd7ac16.png)

![bild](https://user-images.githubusercontent.com/25547797/50358961-5489f180-055b-11e9-9e5e-3993792581fb.png)

![bild](https://user-images.githubusercontent.com/25547797/50358970-594ea580-055b-11e9-8c14-1da21ca3e0f4.png)

![bild](https://user-images.githubusercontent.com/25547797/50358974-5e135980-055b-11e9-90fe-08e9b3d42e03.png)

![bild](https://user-images.githubusercontent.com/25547797/50358980-6370a400-055b-11e9-9388-8851c3a41003.png)

![bild](https://user-images.githubusercontent.com/25547797/50358985-6a97b200-055b-11e9-9b86-ace2834df2c4.png)

![bild](https://user-images.githubusercontent.com/25547797/50358993-6f5c6600-055b-11e9-8d3a-d8367010ef59.png)



------------
Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items as you complete them:

- [ ] A screenshot of the model(s) aligned with KiCad footprints is *very useful*
- [ ] Was the 3D model(s) created in a mechanical modelling program?
- [ ] Source files (or scripts) have been contributed 
- [ ] Step model submitted in 1:1 scale
- [ ] WRL model submitted in 1:2.54 scale
- [ ] 3D model alignment checked against KiCad footprint
- [ ] WRL model uses correct material properties
- [ ] Geometry check performed against mechanical models

Misca1234 commented:
Will be re pushed in combination with a new foot print push

Misca1234 commented:
Due to duplication of 
SIP-3_4.30x1.60mm_P1.30mm.step
SIP-3_4.30x1.60mm_P1.30mm.wrl
These have been removed from this push due to duplicate in 
https://github.com/KiCad/kicad-footprints/pull/1072


They will be renamed and moved to 
Sensor_Current.pretty

Misca1234 commented:
Sanyo_STK4xx-15_59.2x8.0mm_P2.54mm
I think the foot print is wrong

> SIP-9_21.54x3mm_P2.54mm: Can you add this as well? :D

the foot print does not have a data sheet, shall I base it on the SIP-8_19x3mm_P2.54mm ?

After consideration with myself I have reached the conclusion that it is nothign to worry about those "not sure about height." ;)

Shackmeister commented:
Alrighty, suppose we can have a look at the footprint another day :)

I think basing it on SIP-8_19x3mm_P2.54mm  should be fine

Misca1234 commented:
I add have SIP-9_21.54x3mm_P2.54mm

![bild](https://user-images.githubusercontent.com/25547797/55668880-17c6d980-5870-11e9-93a4-cde7c0b4f534.png)

antoniovazquezblanco commented:
What is the status of this PR?

Misca1234 commented:
It is good to go from the contributors point of view
Shackmeister asked for the SIP-9_21.54x3mm_P2.54mm model but I guess he have not review it yet.

Shackmeister commented:
Hi @Misca1234 

Sorry for the long waiting time, I'm trying hard to find some librarian time again.
Checked the models again, and have a few comments.
- It appears all packages with uneven pinnumbers are shifted on the X axis, This at least applies to:
Sanyo_STK4xx-15_59.2x8.0mm_P2.54mm, Sanyo_STK4xx-15_78.0x8.0mm_P2.54mm, SIP-9_21.54x3mm_P2.54mm,

- Making all the pin 1 indicator the "light brown label" color, would make them much more visible.
 
- All the internal files names are not aligned to the actual model name

Misca1234 commented:
@Shackmeister 

Fixed the non-alignment for uneven pins
Fixed internal file names

I did not change the pin 1 marked, need some additional work, not time at the moment.

chmorgan commented:
Hi @Misca1234. Is this one ready for review? If so I can pick it up asap.
